<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPodaci</class>
 <widget class="QDialog" name="frmPodaci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Podaci</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblIme">
         <property name="text">
          <string>Ime:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtIme">
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="placeholderText">
          <string>Ime</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblPrezime">
         <property name="text">
          <string>Prezime:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtPrezime">
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="placeholderText">
          <string>Prezime</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblPol">
         <property name="text">
          <string>Pol:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbPol"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblJmbg">
         <property name="text">
          <string>JMBG:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtJmbg">
         <property name="maxLength">
          <number>13</number>
         </property>
         <property name="placeholderText">
          <string>JMBG</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblDatumRodjenja">
         <property name="text">
          <string>Datum rodjenja:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDateEdit" name="deDatumRodjenja">
         <property name="displayFormat">
          <string>dd/mm/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblAdresa">
         <property name="text">
          <string>Adresa:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="txtAdresa">
         <property name="maxLength">
          <number>100</number>
         </property>
         <property name="placeholderText">
          <string>Adresa</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblNacionalnost">
         <property name="text">
          <string>Nacionalnost:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cbNacionalnost">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblBracnoStanje">
         <property name="text">
          <string>Bracno stanje:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="cbBracnoStanje">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblSlika">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblOstaliPodaci">
     <property name="text">
      <string>Ostali podaci:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtOstaliPodaci"/>
   </item>
   <item>
    <widget class="QLabel" name="lblMisljenje">
     <property name="text">
      <string>Misljenje:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtMisljenje"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>frmPodaci</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>frmPodaci</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
